ow_lander/scripts/dig_circular_action_server.py

```diff
@@ -112,7 +112,7 @@ def now_from_start(start):
 
            self._update_feedback()
            
-        success = self.trajectory_async_executer.wait()
+        success = self.trajectory_async_executer.success() & self.trajectory_async_executer.wait()        
```

Comment:
```suggestion
        success = self.trajectory_async_executer.success() and self.trajectory_async_executer.wait()
```

Reply:
oops yea been doing cpp for too long lol, will adjust!